File: src/main/kotlin/Java/Tareas/Tarea.java
package Java.Tareas;

class Tarea {
    private String nombre;
    private boolean completada;

    public Tarea(String nombre) {
        this.nombre = nombre;
        this.completada = false;
    }

    public String getNombre() {
        return nombre;
    }

    public boolean isCompletada() {
        return completada;
    }

    public void marcarComoCompletada() {
        this.completada = true;
        System.out.println("La tarea '" + nombre + "' ha sido marcada como completada.");
    }

    @Override
    public String toString() {
        return nombre + (completada ? " (completada)" : " (pendiente)");
    }
}

File: src/main/kotlin/kotlinOPP/Tareas/Tarea.kt
package kotlinOPP.Tareas

data class Tarea(val nombre: String, var completada: Boolean = false) {
    fun marcarComoCompletada() {
        completada = true
        println("La tarea '$nombre' ha sido marcada como completada.")
    }

    override fun toString(): String {
        return "$nombre (estado: ${if (completada) "completada" else "pendiente"})"
    }
}